core/tests/protractor_utils/editor.js

```diff
@@ -1316,6 +1316,24 @@ var revertToVersion = function(version) {
   });
 };
 
+var readFeedbackMessages = function() {
+  element(by.css('.protractor-test-exploration-view-feedback-link')).
+    click();
+  var feedbackRowClassName = '.protractor-test-oppia-feedback-tab-row';
+  var feedbacks = element.all(by.css(feedbackRowClassName));
+
+  expect(feedbacks.count()).toEqual(1);
```

Comment:
This is odd. It's not going to be the case in general. Shouldn't this function return an array containing each of the feedback messages, instead?